aiidalab_widgets_base/computational_resources.py

```diff
@@ -1302,6 +1302,10 @@ def _observe_code_setup(self, _=None):
         for key, value in self.code_setup.items():
             if hasattr(self, key):
                 if key == "default_calc_job_plugin":
+                    if "None" in value:
```

Comment:
What is the type of `value`? Somewhat surprising if it would be list for tuple?

Reply:
Ah, okay, that definitely needs an explanatory comment in the code, because `"None" in value` looks suuuper weird :smile: